docker/Dockerfile

```diff
@@ -36,9 +36,6 @@ RUN echo "Adding jupyter lab extensions" \
   && jupyter labextension install --no-build ipycanvas@v0.3.4 \
   && jupyter labextension install --no-build jupyterlab_iframe@0.2.1 \
   && jupyter lab build \
-  && jupyter serverextension enable --py jupyterlab_code_formatter --sys-prefix \
-  && jupyter serverextension enable --py --system jupyterlab_iframe \
-  && jupyter labextension list \
```

Comment:
Can we please keep this line? it's useful for identifying what extensions are installed in build logs.

Reply:
Fixed